CovidGuardianClient/lib/PathHelpers.js

```diff
@@ -0,0 +1,17 @@
+
+// return true if a = b ± padding
+export function fuzzyEqual(a, b, padding) {
+  return (a <= b && b - Math.abs(padding) <= a) || (a > b && b + Math.abs(padding) >= a);
+}
+
+// return distance in km between lat/long points
+// credit to https://stackoverflow.com/questions/27928/calculate-distance-between-two-latitude-longitude-points-haversine-formula
+export function distance(lat1, lon1, lat2, lon2) {
```

Comment:
Checked this with a few examples plus difference from the method used by [google maps api ](https://cloud.google.com/blog/products/maps-platform/how-calculate-distances-map-maps-javascript-api) (below 0.01 meters difference for around 1km distance), so seems perfectly fine!